source/extensions/extensions_build_config.bzl

```diff
@@ -0,0 +1,8 @@
+# FIXFIX comments/docs
+ENVOY_BUILD_CONFIG = {
+    # FIXFIX comments
+    "envoy.filters.http.lua": {
+        "enabled": True,
+        "path": "//source/extensions/filters/http/lua:config",
```

Comment:
You'll want `"path": "@envoy//source/extensions/filters/http/lua:config",` so this will work the same no matter which workspace it gets loaded from.

Reply:
So should I kill repository entirely from all_repositories.bzl? I still don't fully understand when that is required.